### ISSUE TITLE
Request - Button drive skip interface

### ISSUE THREAD

**Issue by cuddawudda** [TortugaPower/BookPlayer#306]
Appreciate that this deviates from the interface design of the app. Perhaps it could be optional?

I regularly listen to audiobooks while taking part in physical activity. To skip using the current interface is difficult to accomplish while running / working etc, and slow - especially if you need to skip significant sections.
Bookmobile does this very well, allowing you to skip forward and back a small # and a large #, either in seconds or minutes. So 5 buttons: - 2 skip forward, 2 back,  a toggle between mins and secs.
I hope you are not immediately put off the idea by looking at the bookmobile interface... but i am sure with the design flair shown in this app you could make it work. But key would be making the buttons 
**LARGE**.

PS, this could also be used for a "driving view"

**Comment by GianniCarlo** [comment]
I'm all for having a 'driving view' like Audible does, simplified controls and big buttons for easier manipulation when most of your attention is required elsewhere. What do you think @pichfl ?

**Comment by pichfl** [comment]
I think that could be quite interesting.

**Comment by cuddawudda** [comment]
Hi pichfi. Wouldnt adding a multiplier be more complicated for the user rather than just selecting the actual numbers? 

Would you consider an option to toggle between second and minutes? With that option you could quickly skip across large sections of the book accurately.

**Comment by pichfl** [comment]
Either multiplier or second set of values, both work for me.

Having a button that changes the interface in function but not in space brings complexity where none is necessary. 

The goal for this view would be to be as easy to use without properly looking or reading the interface, just touching the right area. If the same button does two different but similar things you have to look at a state first before making the interaction. 

So having a set of buttons which skip by amount x in mode 1 and amount y in mode 2 makes for a much more complicated interface than having two sets of buttons, if the main intend is to have an interface that can be used without looking. 

TLDR; No. 

Updated interactions:
- Title information, small artwork, non-interactive progress bar + labels
- Play/Pause
- Skip Forward/Back (ie. 30s / 45s)
- Skip More Forward/Back in larger steps (ie. 5m / 5min)
- Return to regular interface

In the settings screen we would add additional settings for the larger skip values.

PS: My name is "pichfl" with an small "L" at the end.
